src/MonthPickerInput.vue

```diff
@@ -1,78 +1,83 @@
 <template>
   <div
-    class="month-picker-input-container"
-    v-click-outside="hide"
+          class="month-picker-input-container"
```

Comment:
The indent looks a bit heavy here. Could it be decreased?

Reply:
Sure, I will make these changes